src/h2load.cc

```diff
@@ -1238,6 +1304,23 @@ int main(int argc, char **argv) {
               << "cores." << std::endl;
   }
 
+  if (config.nconns < 0) {
+    std::cerr << "-C: the total number of connections made "
+              << "cannot be negative." << std::endl;
+    exit(EXIT_FAILURE);
+  }
+
+  if (config.rate < 0) {
+    std::cerr << "-r: the rate at which connections are made "
+              << "cannot be negative." << std::endl;
+    exit(EXIT_FAILURE);
+  }
+
+  if (config.rate != 0 && config.nthreads != 1) {
+    std::cerr << "-r, -t: warning: the -t option will be ignored when the -r "
+              << "option is in use." << std::endl;
```

Comment:
> As a first attempt, I decided to ignore the -t option. I do agree that adding something similar to the -t option would be nice.

I'm fine wit that.  Current patch is clean enough to add -t support later easily.